### ISSUE TITLE
[dev.icinga.com #11883] Delete action on comments and downtimes in list views not accessible because they lack context

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#2448]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/11883

**Created by elippmann on 2016-06-02 11:56:58 +00:00**

Assignee: _(none)_
Status: _Resolved_  (closed on _2016-06-02 15:25:04 +00:00_)
Target Version: _2.3.4_
Last Update: _2016-06-02 15:25:04 +00:00 (in Redmine)_

---

Screen reader software just reads out "Delete this comment" w/o referencing to the comment. Screen reader software should read out the comment/downtime automatically.

**Changesets**

_2016-06-02 15:23:38 +00:00_ by _elippmann_ 4c2b7f43772b666a9ca62d5fc0f6c555180ec85f

    Make delete action in the comment and dowtime list view accessible
    
    fixes #11883
